parser/prism/Translator.cc

```diff
@@ -1348,12 +1348,38 @@ unique_ptr<parser::Node> Translator::patternTranslate(pm_node_t *node) {
         case PM_IN_NODE: { // An `in` pattern such as in a `case` statement, or as a standalone expression.
             auto inNode = reinterpret_cast<pm_in_node *>(node);
 
-            auto sorbetPattern = patternTranslate(inNode->pattern);
-
+            auto prismPattern = move(inNode->pattern);
```

Comment:
Moving a regular pointer (`pm_node_t *` in this case) like this doesn't do anything. They don't have any special moving semantics, so `std::move`ing them just causes a copy, which is the default anyway, as if you hadn't moved it at all.

```suggestion
            auto prismPattern = inNode->pattern;
```